star-ros2/src/star_spi_bridge/test/test_spi_driver.cpp

```
@@ -49,10 +49,10 @@ TEST_F(SpiDriverTest, FrameDecoding)
   // We can't easily test decode without a working encode or pre-calculated frame
   // For Red Phase, we just assert that decode fails on garbage
   std::vector<uint8_t> garbage = {0x00, 0x00, 0x00};
-  uint16_t seq;
+  uint16_t seq_;
```

Clang (14.0.6)

[warning] variable 'seq_' is not initialized

(cppcoreguidelines-init-variables)